grisera_api/registered_channel/registered_channel_service_graphdb.py

```diff
@@ -18,8 +18,11 @@ class RegisteredChannelServiceGraphDB(RegisteredChannelService):
     registered_data_service (RegisteredDataService): Service to send registered data requests
     """
     graph_api_service = GraphApiService()
-    channel_service = ChannelServiceGraphDB()
-    registered_data_service = RegisteredDataServiceGraphDB()
+
+    def __init__(self):
+        self.channel_service = None
+        self.registered_data_service = None
+        self.recording_service = None
```

Comment:
Missing type definition of these services

Reply:
As above